### ISSUE TITLE
1.19.3 incompatibility?

### ISSUE THREAD

**Issue by SlimeDog** [gamerover98/NoBlackSky#5]
All references to NoBlackSky in console/log:
```
[08:32:42] [Server thread/INFO]: [NoBlackSky] Loading NoBlackSky v2.7
[08:32:42] [Server thread/ERROR]: [NoBlackSky] Please report this error on the issues page on GitHub or Spigot forum
[08:33:33] [Server thread/INFO]: [NoBlackSky] Enabling NoBlackSky v2.7
[08:33:33] [Server thread/WARN]: [NoBlackSky] Disabling NoBlackSky - 2.7
```

**Comment by gamerover98** [comment]
Hello, thanks for your report.
You forgot to add the following error stack trace: 
```JAVA
it.gamerover.nfps.reflection.ReflectionException: net.minecraft.MinecraftVersion.getReleaseTarget() method
        at it.gamerover.nfps.reflection.minecraft.MCMinecraftVersion.<init>(MCMinecraftVersion.java:63) ~[?:?]
        at it.gamerover.nfps.reflection.ReflectionContainer$Minecraft.<init>(ReflectionContainer.java:85) ~[?:?]
        at it.gamerover.nfps.reflection.ReflectionContainer$Minecraft.<init>(ReflectionContainer.java:59) ~[?:?]
        at it.gamerover.nfps.reflection.ReflectionContainer.<init>(ReflectionContainer.java:42) ~[?:?]
        at it.gamerover.nfps.CoreHandler.init(CoreHandler.java:148) ~[?:?]
        at it.gamerover.nfps.NoFootstepSound.onLoad(NoFootstepSound.java:45) ~[?:?]
        at org.bukkit.craftbukkit.v1_19_R2.CraftServer.loadPlugins(CraftServer.java:433) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:219) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:973) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:301) ~[spigot-1.19.3-R0.1-SNAPSHOT.jar:3616-Spigot-d90018e-eecb4c0]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: it.gamerover.nfps.reflection.ReflectionException: Cannot find any method called 'getReleaseTarget' in net.minecraft.MinecraftVersion class
        at it.gamerover.nfps.reflection.Reflection.getMethod(Reflection.java:115) ~[?:?]
        at it.gamerover.nfps.reflection.minecraft.MCMinecraftVersion.<init>(MCMinecraftVersion.java:56) ~[?:?]
        ... 10 more
```

Anyway, I'm working on it.

**Comment by SlimeDog** [comment]
The stack trace does not appear in my console log. I would have included it, if it had.

**Comment by gamerover98** [comment]
Fixed: https://github.com/gamerover98/NoBlackSky/commit/4f878df1ce685c0633142f6ec2cf7e4a6b092e16

**Comment by SlimeDog** [comment]
Confirmed fixed on Spigot 1.19.3. Thanks.
